client/homebrew/editor/editor.jsx

```diff
@@ -162,15 +162,15 @@ const Editor = createClass({
 
 						// Highlight injectors {style}
 						if(line.includes('{') && line.includes('}')){
-							const regex = /(?:^|[^{\n])({(?=((?::(?:"[\w,\-()#%. ]*"|[\w\-()#%.]*)|[^"':{}\s]*)*))\2})/gm;
+							const regex = /(?:^|[^{\n])({(?=((?::(?:"[\w,\-()#%. ]*"|[\w\-()#%.]*)|[^"':{}\n\r\t\f]*)*))\2})/gm;
```

Comment:
Can you update these highlighting regex to match your current regex approach from `markdown.js`? i.e. revert back to `\s` and put the `[:=]` as needed.

Perhaps you can use this example to make sure the highlighting is correct (screenshot shows current behavior; I drew lines where the color is not right):

```
{{test,attribute=myAttr,attribute2="my Attr" text and text}}

{{text,style:myStyle,style2:"my Style" text and text}}

{{text,style:myStyle,attribute=myAttr,style2:"my Style",attribute2="my Attr" text and text}}
```

![image](https://github.com/naturalcrit/homebrewery/assets/5878534/30c74181-1ff2-4a1a-9214-02637caabc33)

Reply:
This should be corrected.